webapp/portlet/src/main/webapp/WEB-INF/jsp/bannerLogo.jsp

```diff
@@ -71,6 +89,9 @@
   });
   <%} %>
   let params = {
+    id: `<%=spaceId%>`,
+    isFavorite: `<%=isFavorite%>`,
+    isMember: `<%=isMember%>`,
```

Comment:
Can non member access to space logo banner? I think it's an unnecessary check at this stage